src/app/static/src/tests/components/runReport/runningReportDetails.test.ts

```diff
@@ -182,20 +183,20 @@ describe(`runningReportDetails`, () => {
             //second time
             jest.runOnlyPendingTimers();
             realSetTimeout(() => {
-                expect(mockAxios.history.get.length).toBe(2);
+                expect(mockAxios.history.get.filter(g => g.url === url).length).toBe(2);
```

Comment:
For some reason the tests with the jest fake timer functions seems to be a bit leaky, even though everything should be reset between tests - I was finding that the second test which invoked this code was finding a leftover axios call from the previous test. Some race condition, but I wasn't able to clear it, so filter on expected url - which differs between the two tests - here.